### DISCUSSION TITLE
Is it possible to export only the currently filtered items?

### DISCUSSION THREAD

**Discussion by tomwitt2** [Komet/MediaElch#1391] [Q&A] [answered]
When I have a filtered set of movies (4 out of 91), and I export to CSV, all 91 movies are exported instead of the 4 I have filtered.  Is there a way to limit the export to the currently filtered items?

**Comment by bugwelle** [comment, marked as answer]
Hi,

there is currently no such feature. The issue I see is that most of the time, the user has selected one movie. I see two options:

1. Export selected movies only if more than 1 is selected
2. Add either another drop-down item "Export selected movies" or a checkbox "Export selected movies only"

I will need to see how much work that would be.

Regards,
Andre
